tests/test_drs.py

```diff
@@ -6,10 +6,23 @@
 from tests.test_bundles import get_bundle_doc
 
 
-def generate_presigned_url_response(did, protocol="", status=200):
-    full_url = (
-        "https://fictitious-commons.io/data/download/" + did + "?protocol=" + protocol
-    )
+def generate_presigned_url_response(did, protocol="", query_string="", status=200):
```

Comment:
minor nitpick: protocol ends up in the query string already, so `query_string` is really like `additional_query_param` and it might be better for this function to accept them as a dictionary and then dump them into the right format so the caller doesn't have to query whether or not to include ? or &. basically, standardize this function as `func(did, params, status)`